Portal/src/Datahub.Portal/Pages/Tools/Users/UsersPage.razor

```diff
@@ -0,0 +1,41 @@
+@page "/tools/users"
+
+@inject IDbContextFactory<DatahubProjectDBContext> _dbContextFactory
+@attribute [Authorize(Roles = "DHPGLIST")]
```

Comment:
Glad you put the Auth in but could you wrap it with the `DatahubAuthView` component instead? Just so that all our role access is in a single file

Reply:
bump @nicksalt

Reply:
Fixed in commit d93b5b6